packages/devtools_app/test/hover_test.dart

```diff
@@ -9,12 +9,15 @@ import 'package:test/test.dart';
 const _defaultStyle = TextStyle();
 const _textSpan = TextSpan(children: [
   TextSpan(text: 'hello'),
```

Comment:
bonus points: make this text use a variable width font to ensure it is robust.

Reply:
Done.